### PR TITLE
expression/agg: add unit-test.

### PR DESCRIPTION
**Pull request by winoros**
i'll add bench test in later pr.
This pr improve the test coverage to 66%.
PTAL @zz-jason @XuHuaiyu

### CODE REVIEW

**Comment by shenli** [comment]
Please fix the CI.

**Comment by shenli** [comment]
Please add a few comments in the test file.

**Comment by zz-jason** [comment]
what's the test coverage now ?

**Comment by zz-jason** [comment]
LGTM

File: expression/aggregation/aggregation_test.go [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/ast"
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/mysql"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/mock"
+)
+
+var _ = Suite(&testAggFuncSuit{})
```
Comment (shenli) [inline expression/aggregation/aggregation_test.go]: There should be only one test Suite in a package.

Comment (shenli) [reply expression/aggregation/aggregation_test.go]: Ignore this comment.

**Comment by shenli** [comment]
LGTM

**Pull request merged**